docs/using-the-jdbc-driver/using-plugins/UsingTheFailoverPlugin.md

```diff
@@ -21,15 +21,16 @@ The failover plugin will be enabled by default if the [`wrapperPlugins`](../Usin
+|----------------------------------------|:-------:|:--------------------------------------------------------------------------------:|:-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `failoverMode`                         | String  |                                        No                                        | Defines a mode for failover process. Failover process may prioritize nodes with different roles and connect to them. Possible values: <br><br>- `strict-writer` - Failover process follows writer node and connects to a new writer when it changes.<br>- `reader-or-writer` - During failover the driver tries to connect to any available/accessing reader node. If no reader is available, the driver can connect to a writer node. This logic mimics the logic of the Aurora read-only cluster endpoint.<br>- `strict-reader` - The driver tries to connect to any available reader node. If no reader is available, the driver raises an error. Reader failover to a writer node will only be allowed for single-node clusters. This logic mimics the logic of the Aurora read-only cluster endpoint.<br><br>If this parameter is omitted, default value depends on connection url. For Aurora read-only cluster endpoint, it's set to `reader-or-writer`. Otherwise, it's `strict-writer`. | Default value depends on connection url. For Aurora read-only cluster endpoint, it's set to `reader-or-writer`. Otherwise, it's `strict-writer`.                                                     |
```

Comment:
"`reader-or-writer` - During failover the driver tries to connect to any available/accessing reader node."
Should `accessing` instead be `accessible`? 

"If no reader is available, the driver can connect to a writer node."
Should change `can` to `will`. 

"`strict-reader` - The driver tries to connect to any available reader node...."
Might be good to begin this something like "During failover, the driver....".